CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### Fixes
 
 - Fix for issue where long-running python models led to invalid session errors ([544](https://github.com/databricks/dbt-databricks/pull/544))
```

Comment:
You can delete my previous line, since your commit is a much more complete solution.